### PR TITLE
Update samples to use a computed variable for head

### PR DESCRIPTION
**Pull request by hmsk**
- [x] Class API
- [x] Options API
- [x] Composition API -> Need to ask @danielroe how to use computed function on `useMeta` (looks not supported yet)

### CODE REVIEW

File: pages/composition-api.vue [hunk 1]
```diff
@@ -81,7 +82,13 @@ export default defineComponent({
...
     })
 
-    useMeta({ title: 'Composition API Demo' })
+    useMeta({
+      title: 'Composition API Demo',
+      meta: [{
+        name: 'message',
+        content: message.value // It's not reactive
+      }]
+    })
```
Comment (danielroe) [inline pages/composition-api.vue:91]: @hmsk Sorry for the delay in replying. I'm afraid this isn't actually that intuitive at the moment. I might need to add the functionality to `@nuxtjs/composition-api`.

Comment (danielroe) [reply pages/composition-api.vue:91]: Does this API look good to you?

https://github.com/nuxt-community/composition-api/pull/255

Comment (hmsk) [reply pages/composition-api.vue:91]: Makes sense for me. `useMeta(() => ({}))` is intuitive and looks similar to the type of `head ()` in options/class API.

**Comment by danielroe** [comment]
@hmsk my proposed api should now be useable in `v0.13.0`.

File: pages/composition-api.vue (hunk 1 above)
Comment (hmsk) [reply pages/composition-api.vue]: a812357 😉

**Comment by hmsk** [comment]
Thanks, @danielroe for your responsive updates on `@nuxt/composition-api`!

**Pull request merged**